File: 01.JS-Basic/По сложни проверки Упражнение/HotelRoom.js
function hotelRoom(input) {

    let mounth = input[0]
    let nightNum = Number(input[1])

    let priceApartment = 0.0
    let priceStudio = 0.0

    if (mounth === "May" || mounth === "October") {
        if (nightNum > 14) {
            priceStudio = 50 * 0.7
            priceApartment = 65 * 0.9
        }
        else if (nightNum > 7) {
            priceStudio = 50 * 0.95
            priceApartment = 65
        }
        else {
            priceStudio = 50
            priceApartment = 65
        }
    }
    else if (mounth === "June" || mounth === "September") {
        if (nightNum > 14) {
            priceStudio = 75.2 * 0.8
            priceApartment = 68.7 * 0.9
        }
        else {
            priceStudio = 75.2
            priceApartment = 68.7
        }
    }
    else if (mounth === "July" || mounth === "August") {
        if (nightNum > 14) {
            priceStudio = 76
            priceApartment = 77 * 0.9
        }
        else {
            priceStudio = 76
            priceApartment = 77
        }
    }
    console.log(`Apartment: ${(nightNum * priceApartment).toFixed(2)} lv.`)
    console.log(`Studio: ${(nightNum * priceStudio).toFixed(2)} lv.`)

}

hotelRoom(["June","14"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatementsAdvanced/Exercise/HotelRoom.java
package ConditionalStatementsAdvanced.Exercise;

import java.util.Scanner;

public class HotelRoom {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String month = scan.nextLine();
        int days = Integer.parseInt(scan.nextLine());
        double apartmentPrice = 0.0;
        double studioPrice = 0.0;

        switch (month) {
            case "May":
            case "October":
                studioPrice = 50.00;
                apartmentPrice = 65.00;
                if (days > 14) {
                    studioPrice *= 0.7;
                } else if (days > 7) {
                    studioPrice *= 0.95;
                }
                break;
            case "June":
            case "September":
                studioPrice = 75.20;
                apartmentPrice = 68.70;
                if (days > 14) {
                    studioPrice *= 0.8;
                }
                break;
            case "July":
            case "August":
                studioPrice = 76.00;
                apartmentPrice = 77.00;
                break;
        }
        if (days > 14) {
            apartmentPrice *= 0.9;
        }
        System.out.printf("Apartment: %.2f lv.\n", apartmentPrice * days);
        System.out.printf("Studio: %.2f lv.", studioPrice * days);
    }
}
